internal/pipeline/executor.go

```diff
@@ -139,15 +142,32 @@ func (p *pkg) resolveSources() []string {
 			// include only this pkg sources
 			pkgPaths = append(pkgPaths, p.Path())
 		case sourceAllSubPkgs:
-			// TODO(droot): figure out how to enumerate sub packages for current pkgPath.
-			// For now including current pkg
+			// For now including current pkg and subpkgs in current directory
```

Comment:
nit: remove `For now` because `sourceAllSubPkgs` requires us to include current pkg and subpkgs.

Reply:
done